queries.sql

```diff
@@ -98,4 +98,48 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 
 SELECT species, MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-1-1' AND '2000-12-31' GROUP BY species;
+
+___________________________________________________________________________________
+
+
+SELECT animals.name, owners.full_name 
+FROM animals 
+JOIN owners 
+ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Melody';
```

Comment:
- [ ] Good job trying to make some of changes mentioned by the previous reviewer. However, as he mentioned and as required for the project requirements, kindly make sure you update the above query by using the owner's full_name i.e `... WHERE owners.full_name = 'Melody Pond'` instead of `... WHERE owners.full_name = 'Melody'`.